pom.xml

```diff
@@ -70,7 +70,7 @@
     <dependency>
       <groupId>org.fusesource.jansi</groupId>
       <artifactId>jansi</artifactId>
-      <version>2.0.1</version>
+      <version>2.2.0</version>
       <optional>true</optional>
```

Comment:
Why is this marked optional instead of simply declaring it? It feels like unnecessary complexity. The model code checks for its presence but the test code doesn't seem to.

Reply:
It must be optional, it is Maven that's responsible to provide it and to decide which features are supported.
Maven 3.3.9 and earlier would otherwise have mixed output.